SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -426,13 +426,18 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 		PlayerSpawner = NewObject<USpatialPlayerSpawner>();
 		SnapshotManager = MakeUnique<SpatialSnapshotManager>();
 
-		if (GetDefault<USpatialGDKSettings>()->bAsyncLoadNewClassesOnEntityCheckout)
+		if (SpatialSettings->bAsyncLoadNewClassesOnEntityCheckout)
 		{
 			AsyncPackageLoadFilter = NewObject<UAsyncPackageLoadFilter>();
 			AsyncPackageLoadFilter->Init(
 				FOnPackageLoadedForEntity::CreateUObject(this, &USpatialNetDriver::OnAsyncPackageLoadFilterComplete));
 		}
 
+		if (SpatialSettings->bEnableInitialOnlyReplicationCondition && !IsServer())
+		{
+			InitialOnlyFilter = MakeUnique<SpatialGDK::InitialOnlyFilter>(this);
```

Comment:
Does !IsServer prevent InitialOnly being applied to server workers? It's kind of a weird space but since we support server->server replication is it also worth thinking about initialOnly here too?

Reply:
This doesn't really make sense, either you have interest from being authoritive or interest from being always interested (where initialonly doesn't make sense). Perhaps matters a little at borders but sounds like an edge case that's not worth worrying about

Reply:
This is only setup on clients because server bypass the initial only filter - per description -
`Non-auth servers continue to receive this initial only data unfiltered.`